perfkitbenchmarker/scripts/object_storage_api_tests.py

```diff
@@ -271,81 +262,238 @@ def PercentileCalculator(numbers):
   return result
 
 
-def _ListObjects(storage_schema, bucket, prefix, host_to_connect=None):
-  """List objects under a bucket given a prefix.
+# ### Backends ###
 
-  Args:
-    storage_schema: The address schema identifying a storage. e.g., "gs"
-    bucket: Name of the bucket.
-    prefix: A prefix to list from.
-    host_to_connect: An optional endpoint string to connect to.
+class ObjectStorageBackend(object):
```

Comment:
However, I am happy to move everything to one directory right now. Until we have the `ObjectStorageService` objects, it would be much easier that way.